rust/ql/src/queries/security/CWE-295/DisabledCertificateCheck.qhelp

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+The <code>danger_accept_invalid_certs</code> and <code>danger_accept_invalid_hostnames</code> options on TLS connectors and HTTP clients control whether certificate and hostname verification are performed. If set to <code>true</code>, the client will accept any certificate or any host name, making it susceptible to man-in-the-middle attacks.
```

Comment:
Also, this sentence reads oddly because it's ambiguous what is set to true: either of these options, both options, or the client?

`If set to true, the client will accept any certificate or any host name, making it susceptible to man-in-the-middle attacks.`

Might be worth rewording for clarity.